packages/modules/vehicles/smarteq/api.py

```diff
@@ -355,73 +698,432 @@ def get_status(self, vin: str) -> int:
                 soc = -1
                 range = 0.0
 
-        except Exception:
-            log.exception('get_status')
-            self.log.debug("get_status: result:\n" + res_json)
+        except requests.exceptions.HTTPError as e:
+            self.log.warning("get_status: authentication by access_token failed: " + str(e.response.status_code))
+            self.log.debug("get_status HTTPError: " + str(e.response.status_code) + " - " + str(e.response.text))
+        except Exception as e:
+            self.log.warning("get_status: authentication by access_token failed: " + str(e.response.status_code))
+            self.log.debug("get_status Exception: " + str(e.response.status_code) + " - " + str(e.response.text))
+            res = json.loads(e.response.text)
```

Comment:
Die nested_key_exists-Methode wird in dieser Datei und in socutils.py definiert. Mehr im Python-Style ist es, hier noch ein except KeyError anzufügen und die Exception mit einer entsprechenden Meldung fürs Frontend zu werfen.